python/src/opendp/extras/polars/__init__.py

```diff
@@ -44,6 +44,53 @@ class DPExpr(object):
     >>> import polars as pl
     >>> pl.len().dp
     <opendp.extras.polars.DPExpr object at ...>
+
+    In addition to the DP-specific methods documented below, many Polars ``Expr`` methods are also supported.
+    For these, the best documentation is the official Polars documentation.
+
+    .. list-table:: Supported Polars ``Expr`` Methods
+        :header-rows: 1
+
+        * - Method
+          - Operators
+          - Use for
+          - Usage
```

Comment:
To save space, how would you feel about condensing into two columns:

* method (with link)
* notes

Where for the most part, notes talks about privacy-specific behavior, but if space allows, also some pointers. 

Some suggestions:

# Context Transformations

.
* with_columns
* expressions must not reorder or add/remove rows

.
* filter
* margin descriptors may be lost; consider group_by instead

# Context Aggregations

.
* select
* can specify multiple expressions

.
* group_by/agg
* can specify multiple grouping keys and expressions



# Expression Transformations
.
* eq, ne, lt, le, gt, ge
* Or use operators: ``==`` ``!=`` ``<`` ``<=`` ``>`` ``>=``

.
* clip
* establishes bounds descriptor

.
* fill_null
* if fill value is non-null, establishes non-nullity descriptor

.
* fill_nan
* if fill value is non-null and non-nan, establishes non-nan descriptor

# Expression Aggregators
.
* len
* number of rows, including nulls

.
* count
* number of rows, not including nulls

.
* n_unique
* number of unique rows

I think code examples are already covered by Polars. Please don't hesitate to disagree if you feel differently!

Reply:
Thanks. Should be addressed.